VideoLocker/assets/config/registration_form.json

```diff
@@ -35,7 +33,6 @@
         "errorMessages": {},
         "placeholder": "",
         "defaultValue": "",
```

Comment:
Yep, just checking, because I believe that that help text was added deliberately to address the pain point of learners not understanding which name was used where, and not registering with a name they wanted on their certs. It is also important to communicate that usernames once chosen, can't be changed. I defer to @marcotuts if this was a Product call, I just was not aware of the decision. Thanks!